andela/wg-silicon
# [Feature #158626109] Change how user adds meals

### What does this PR do?
Changes how the user adds meals

### Description of Task to be completed?
- The user can now add time, ingredient name, amount and weight at once when adding a new meal to the nutrition plan.
- Previously, a user had to add the time first, before adding the other meal properties, which was tiring.

### How should this be manually tested?
1. Visit [wger application](https://wg-silicon.herokuapp.com/en/software/features). 
2. Under the `Nutrition` dropdown, select `Nutrition plans`.
3. Click the green `Add Nutrition plan` button.
4. Click the grey `No meals for this plan. Add one now.` button.
5. Note that one can add time, ingredient name, amount and weight at once.


### What are the relevant pivotal tracker stories?
- [Change how user adds meals](https://www.pivotaltracker.com/story/show/158626109)

### Screenshots
- __Before changing how user adds meals__
One had to first add time
<img width="656" alt="screen shot 2018-07-15 at 14 14 44" src="https://user-images.githubusercontent.com/29580769/42733412-a7337676-8839-11e8-9bdf-f676c48a65ae.png">

Then add the meal items
<img width="674" alt="screen shot 2018-07-15 at 14 15 20" src="https://user-images.githubusercontent.com/29580769/42733415-b6dbc4d4-8839-11e8-95f6-ac08f9c04156.png">


- __After changing how user adds meals__
One can now add the time and meal items at one go
<img width="661" alt="screen shot 2018-07-15 at 14 15 37" src="https://user-images.githubusercontent.com/29580769/42733426-cebf8702-8839-11e8-961d-f944a9427650.png">
